VirtoCommerce.CatalogModule.Web.Core/Converters/CategoryConverter.cs

```diff
@@ -39,7 +39,7 @@ public static webModel.Category ToWebModel(this moduleModel.Category category, I
             var parents = new List<moduleModel.Category>();
             if (category.Parents != null)
             {
-                retVal.Outline = string.Join("/", category.Parents.Select(x => x.Id));
+                retVal.Outline = category.Outlines.FirstOrDefault().ToString();
```

Comment:
Previous version always returned physical outline, and new version can return virtual outline - this is a breaking change.
Need to replace Outlines.FirstOrDefault() with the code, that selects the physical outline.